src/schema/nmdc.yaml

```diff
@@ -871,11 +898,41 @@ enums:
         text: EMSL
         title: Environmental Molecular Sciences Laboratory
         meaning: https://ror.org/04rc0xn13
-        comments:
+        notes:
           - replaces Environmental Molecular Science Laboratory
           - replaces Environmental Molecular Sciences Lab
 
 slots:
+  emsl_project_identifier: { }
+  img_identifiers:
+    title: IMG Identifiers
+    description: >-
+      A list of identifiers that relate the biosample to records in the IMG database.
+    multivalued: true
+    todos:
+      - add is_a or mixin modeling
+      - what are the class of IMG records?!
+
+  emsl_biosample_identifiers:
+    title: EMSL Biosample Identifiers
+    description: >-
+      A list of identifiers for the biosample from the EMSL database.  This is
+      used to link the biosample, as modeled by NMDC, to the biosample in the planned EMSL NEXUS database.
+    multivalued: true
+    is_a: sample_identifiers
```

Comment:
Should we rename `sample_identifiers` to `biosample_identifiers`? Very trivial, but since it looks like most of the slots that inherit this are different types of **biosample** identifiers, it can help clear some vagueness around the word _sample_?

Reply:
good point